main.py

```diff
@@ -526,6 +526,10 @@ async def on_message(message):
 OKIB_GATHER_EMOJI_STRING = "<:ib:{}><:ib2:{}>".format(IB_EMOJI_ID, IB2_EMOJI_ID)
 OKIB_GATHER_PLAYERS = 8 # not pointless - sometimes I use this for testing
 
+PUB_HOST_ROLE_ID = 791279611311947796
```

Comment:
ugh, ideally these would go into the `params.py`, but I know that's annoying for you to edit. can change later I guess, whatever.

Reply:
yeah nvm I'll rethink this later, ignore this comment